docs/changes/io.registry/11916.api.rst

```diff
@@ -0,0 +1 @@
+The ``write`` function now returns content results.
```

Comment:
I think this should be a bit elaborated "now is allowed to return possible content results, which means that custom writers could, for instance, create and return an instance of some container class rather than a file on disk."